### ISSUE TITLE
recalculateOffsets not working

### ISSUE THREAD

**Issue by astanciu** [sporritt/jsPlumb#350]
Hi -

I don't know if I'm doing something wrong or if there's something wrong with recalculateOffsets method. 

I have a panel that I want to hide/show when it's title is clicked, and the EndPoints remain floating after hiding the panel's body.
$('#Adapter0 .panel-body').hide()
jsPlumb.recalculateOffsets('Adapter0')

I tried debugging the code to see what happens and the recalculateOffets call goes to  this.getDragManager().updateOffsets(); 
and inside this there is a check:
this.updateOffsets = function (elId) {
            if (elId != null) {

elID is always null for me. Shouldn't the recalculateOffsets method be defined something like:
recalculateOffsets:function(ELEMENT) {
            this.getDragManager().updateOffsets(ELEMENT);
        },

What am I missing here?
Thanks

**Comment by ericnewcomer** [comment]
Just wanted to +1 this. 

I noticed the very same thing when upgrading to the latest jsPlumb (1.7.5). I also made the above change in the jsPlumb source out of curiosity and verifying that our project works as it did previously. Is this an oversight? Or did something change? We are also moving from the jQuery based library to the vanilla dom implementation, so perhaps there's something we need to do differently here.
